client/my-sites/plugins/main.jsx

```diff
@@ -444,27 +463,33 @@ export default React.createClass( {
 	renderPluginList( plugins, header, isWpCom ) {
 		let headerMarkup;
 		const slug = header.replace( / /g, '' );
+		const inBulkEditMode = this.state.bulkManagement.indexOf( isWpCom ? 'wpcom' : 'jetpack' ) !== -1 ? true : false;
```

Comment:
no need for the ternary operator... it could just be `const inBulkEditMode = this.state.bulkManagement.indexOf( isWpCom ? 'wpcom' : 'jetpack' ) > -1`